lib/streamlit/delta_generator.py

```diff
@@ -453,26 +523,39 @@ def _block(self, block_proto=Block_pb2.Block()):
         return block_dg
 
     def beta_expander(self, label=None, expanded=False):
-        """Create a container that can be expanded and collapsed.
+        """Insert a multi-element container that can be expanded/collapsed.
+
+        Inserts a container into your app that can be used to hold multiple elements
+        and can be expanded or collapsed by the user. When collapsed, all that is
+        visible is the provided label.
 
-        Similar to `st.container`, `st.expander` provides a container
-        to add elements to. However, it has the added benefit of being expandable and
-        collapsible. Users will be able to expand and collapse the container that is
-        identifiable with the provided label.
+        To add elements to the returned container, you can use "with" notation
+        (preferred) or just call methods directly on the returned object. See
+        examples below.
 
         Parameters
         ----------
         label : str
-            A short label used as the header for the expander.
-            This will always be displayed even when the container is collapsed.
-        expanded : boolean
-            The default state for the expander.
-            Defaults to False
+            A string to use as the header for the expander.
+        expanded : bool
+            If True, initializes the expander in "expanded" state. Defaults to
+            False (collapsed).
 
         Examples
         --------
-        >>> expander = st.beta_expander("Expand Me")
-        >>> expander.write("I can be expanded")
+        >>> st.line_chart({"data": [1, 5, 2, 6, 2, 1]})
+        >>>
+        >>> with st.beta_expander("See explanation"):
+        ...     st.write(\"\"\"
+        ...         The chart above shows some numbers I picked for you.
+        ...         I rolled actual dice for these, so they're *guaranteed* to
+        ...         be random.
+        ...     \"\"\").
+        ...     st.image("https://static.streamlit.io/examples/dice.jpg")
+
+        .. output ::
+            https://share.streamlit.io/0.66.0-Wnid/index.html?id=T9Xt6z7nkweBSLiJcGTbYV
```

Comment:
General, not sure if the url in the static app automatically controls what version but the demos will likely have to be updated (at least for expander) to get the latest visual changes.

Reply:
Yup, just pushed the update!

(The url in the static app is kind of a misnomer; what actually happens is that you have to statically build the files locally, and push them to an S3 bucket. See [Notion](https://www.notion.so/streamlit/Static-Embedded-Apps-b9e2e63dfadd4d83a58236b125b57b30))